pkg/proxy/util/nodeport_addresses_test.go

```diff
@@ -280,9 +280,9 @@ func TestContainsIPv4Loopback(t *testing.T) {
 			want:        true,
 		},
 		{
-			name:        "all zeros ipv6", // interpret all zeros equal for IPv4 and IPv6 as Golang stdlib
```

Comment:
to be honest I've added this and I can't remember why, maybe because of the port opener? anyway, it seems this is more correct and as you explain there is no need